node/src/effect.rs

```diff
@@ -1676,6 +1663,6 @@ impl<REv> EffectBuilder<REv> {
 #[macro_export]
 macro_rules! fatal {
     ($effect_builder:expr, $($arg:tt)*) => {
-        $effect_builder.fatal(file!(), line!(), format_args!($($arg)*).to_string())
+        $effect_builder.fatal(file!(), line!(), format!($($arg)*).to_string())
```

Comment:
Is the `.to_string()` redundant now?

`format!(…)` is equivalent to `format_args!(…).to_string()`, is it?